python/examples/basic/Schema_Configuration.ipynb

```diff
@@ -800,101 +769,78 @@
+       "col4                  3.0                3.0                5.0   \n",
+       "col5                  NaN                NaN                NaN   \n",
        "\n",
-       "        distribution/q_75  distribution/q_90  distribution/q_95  \\\n",
+       "        distribution/q_90  distribution/q_95  distribution/q_99  \\\n",
        "column                                                            \n",
        "col1                  NaN                NaN                NaN   \n",
        "col2                  5.0                5.0                5.0   \n",
+       "col3                  NaN                NaN                NaN   \n",
        "col4                  5.0                5.0                5.0   \n",
        "col5                  NaN                NaN                NaN   \n",
-       "col3                  NaN                NaN                NaN   \n",
        "\n",
-       "        distribution/q_99  cardinality/est  cardinality/upper_1  \\\n",
-       "column                                                            \n",
-       "col1                  NaN              NaN                  NaN   \n",
-       "col2                  5.0              NaN                  NaN   \n",
-       "col4                  5.0              3.0              3.00015   \n",
-       "col5                  NaN              NaN                  NaN   \n",
-       "col3                  NaN              3.0              3.00015   \n",
+       "        distribution/stddev  cardinality/est  cardinality/lower_1  \\\n",
+       "column                                                              \n",
+       "col1                    NaN              NaN                  NaN   \n",
+       "col2                    1.0              NaN                  NaN   \n",
+       "col3                    NaN              3.0                  3.0   \n",
+       "col4                    1.0              3.0                  3.0   \n",
+       "col5                    NaN              NaN                  NaN   \n",
        "\n",
-       "        cardinality/lower_1  \n",
+       "        cardinality/upper_1  \n",
        "column                       \n",
        "col1                    NaN  \n",
        "col2                    NaN  \n",
-       "col4                    3.0  \n",
-       "col5                    NaN  \n",
-       "col3                    3.0  "
+       "col3                3.00015  \n",
+       "col4                3.00015  \n",
+       "col5                    NaN  "
       ]
      },
-     "execution_count": 12,
+     "execution_count": 8,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
     "import whylogs as why\n",
-    "\n",
+    "from whylogs.core import DatasetSchema\n",
     "df['col5'] = 0\n",
-    "result = why.log(df, schema=MyCustomSchema())\n",
+    "result = why.log(df, schema=DatasetSchema(resolvers=MyCustomResolver()))\n",
```

Comment:
Does a custom DatasetSchema not work anymore?

Reply:
It doesn't.

tbh, I'm not sure why it doesn't work anymore, I just noticed that the current approach ended up using a standard resolver instead of the custom one.
I'll look into it.

Reply:
It looks like it's because DatasetSchema wasn't a Dataclass originally even though it was intended to be. Those changes introduced this new behavior.